integration-tests/contract-increase-paid-storage-operation.spec.ts

```diff
@@ -23,7 +27,25 @@ CONFIGS().forEach(({ lib, rpc, setup, protocol, knownContract }) => {
       expect(op.status).toEqual('applied');
       done();
     });
+    
+    limanetAndAlpha('should be able to increase the paid storage of a contract successfully', async (done) => {
+      const paidSpaceBefore = await client.getStoragePaidSpace(knownContract);
```

Comment:
Ah yes, I forgot that we could have done that instead. 

> Can this test have a side effect if run simultaneously in different PRs? What if 2 runs increase the storage of knownContract at the same time?

That's an interesting thought. It might actually cause a failure in our tests since perhaps it might increment the amount of paid storage by 2. I can see 2 possibilities in this case:
1. We don't check for the incremented amount, and just check whether it's `defined` or `undefined` or,
2. Make sure our tests run sequentially, which might increase our test runtimes by quite a bit

Do you have any other suggestions?

Reply:
We can create a new contract in the test and use this one instead of the known Contract, so it shouldn't have a concurrent effect.

Reply:
ah yes, that's actually a much cleaner solution. Will update